lib/typeprof/core/ast/value.rb

```diff
@@ -287,6 +287,47 @@ def install0(genv)
       end
     end
 
+    class KeywordHashNode < Node
```

Comment:
This is the same behavior as `HashNode`, should I remove this and use `HashNode` instead of?

Reply:
It's the exact same definition, right? Then, let's reuse HashNode for now. We can separate it if needed later